src/main/java/frc/robot/commands/ColorWheelWithJoystick.java

```diff
@@ -0,0 +1,123 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Const;
+import frc.robot.subsystems.Elevator;
+import edu.wpi.first.wpilibj.DriverStation;
+import com.revrobotics.ColorMatch;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorSensorV3;
+
+
+public class ColorWheelWithJoystick extends CommandBase {
+    private final I2C.Port i2cPort = I2C.Port.kOnboard;
+    private final ColorSensorV3 colorSensor = new ColorSensorV3(i2cPort);
+    private final ColorMatch colorMatcher = new ColorMatch();
+    private final Color kBlueTarget = ColorMatch.makeColor(0.143, 0.427, 0.429);
+    private final Color kGreenTarget = ColorMatch.makeColor(0.197, 0.561, 0.240);
+    private final Color kRedTarget = ColorMatch.makeColor(0.561, 0.232, 0.114);
+    private final Color kYellowTarget = ColorMatch.makeColor(0.361, 0.524, 0.113);
+    private String colorString;
+    private boolean colorFound = false;
+    private boolean rotationComplete = false;
+    private Elevator.WheelColor wheelColor;
+
+    edu.wpi.first.wpilibj.PWMVictorSPX pwm = new edu.wpi.first.wpilibj.PWMVictorSPX(0);
+    
+    Color detectedColor = colorSensor.getColor();
+    ColorMatchResult match = colorMatcher.matchClosestColor(detectedColor);
+    String desiredColor;
+    String gameSpecificColor = DriverStation.getInstance().getGameSpecificMessage();
+    Elevator elevator;
+  
+    public ColorWheelWithJoystick() {
+    }
+
+    public void initialize() {
+      switch (gameSpecificColor){
```

Comment:
We are already performing the offset in the elevator `goToColor()` function, you don't need to offset these here.